xonsh/environ.py

```diff
@@ -942,6 +943,20 @@ def DEFAULT_VARS():
             "Note: ``erasedups`` is supported only in sqlite backend).",
             doc_store_as_str=True,
         ),
+        "HOSTNAME": Var(
+            is_string,
+            ensure_string,
+            ensure_string,
+            uname().node,
+            "Automatically set to the name of the current host.",
+        ),
+        "HOSTTYPE": Var(
+            is_string,
+            ensure_string,
+            ensure_string,
+            uname().machine,
```

Comment:
Similarly, this should be `default_value(lambda env: uname.machine)`

Reply:
Done.